src/core_plugins/timelion/server/handlers/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,50 @@
+const chainRunnerFn = require('../chain_runner');
+const tlConfig = require('../../series_functions/__tests__/fixtures/tlConfig.js')();
+
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
+  const chainRunner =  chainRunnerFn(tlConfig);
+
+  async function parse(data) {
+    const request = {
+      sheet: data,
+      time: {
+        from: 'now-15m',
+        to: 'now',
+        mode: 'quick',
+        interval: 'auto',
+        timezone: 'Europe/Berlin'
+      }
+    };
+    return await Promise.all(chainRunner.processRequest(request));
```

Comment:
The difference I see is that in `routes/run.js` the `Promise` is imported from bluebird while this uses `global.Promise`. Maybe there's a difference in the tolerance regarding being passed single promises vs arrays of `Promise.all` between bluebird and native promises. I'm not sure the `Promise.all` is required at all since `chainRunner.processRequest` always seems to return a single promise.